bitcoin_client/tests/README.md

```diff
@@ -0,0 +1 @@
+These tests verify the compatibility layer of the bitcoin_client when ran against the legacy app (v1.6.5) using the LegacyClient.
```

Comment:
Tests needs to be run from the `tests/` directory, that's worth a notice in the `README`, or using absolute paths in order to properly find `JSON` resources.

Reply:
Good suggestion, I instead fixed the path problems and now they are also runnable from `/bitcoin_client`, which is cool to have.
Some work needed to do the same for the main repository tests, but less important in that case. (Also, there are multiple test folders, so not as useful, perhaps).